packages/api/src/wallets/VC.ts

```diff
@@ -101,22 +101,25 @@ export default class VCWallet extends Wallet {
     return this.command<{
       success: boolean;
       transactions: TransactionType[];
-    }>('vc_spend_vc', args);
+    }>('vc_spend', args);
   }
 
   async addVCProofs(proofs: string) {
     return this.command<{
       proofs: any;
-    }>('add_vc_proofs', { proofs });
+    }>('vc_add_proofs', { proofs });
   }
 
   async getProofsForRoot(root: string) {
-    return this.command<{ proofs: any }>('get_proofs_for_root', { root });
+    if (!root) {
+      return { proofs: {} };
+    }
```

Comment:
We should let the backend respond even if the request is malformed. The backend could change how it handles this scenario.